ground/src/main/java/com/google/android/ground/persistence/remote/firebase/protobuf/ModelToProtoExt.kt

```diff
@@ -128,11 +129,16 @@ fun LocationOfInterestMutation.createLoiMessage(user: User) = locationOfInterest
   }
 }
 
-private fun ValueDelta.toMessage() = taskData {
-  val me = this@toMessage
+private fun toSkippedTaskData(id: String) = taskData {
+  // TODO: What should be the ID?
+  taskId = id
+  skipped = true
```

Comment:
We can't know at this point if the task was skipped by the user, or if it's just missing data.

Reply:
Ah, I misinterpreted it based on the proto comments. Thanks for pointing it out.

Reply:
👍  Would you be able to update the comments to make it more explicit? We should make sure to say "explicitly skipped by the user".

Reply:
Done. I'll update the comment separately